inception-kb/src/main/resources/de/tudarmstadt/ukp/inception/kb/knowledgebase-profiles.yaml

```diff
@@ -1,6 +1,7 @@
 british_museum:
   name: British Museum
-  sparql-url: http://collection.britishmuseum.org/sparql
+  type: REMOTE
+  access-url: http://collection.britishmuseum.org/sparql
```

Comment:
Why do you need to change the name here? anything specific?

Reply:
It's more general. For remote KBs it is the sparql endpoint, for local KBs it is the download URL.

Reply:
We could indeed include some "built-in" ontologies which we could import from a classpath URL.

Reply:
@MdeBoer95 As @reckart suggested, it would be a better idea instead to have some built it ontogies on classpath URL. 

@reckart do we see a use case in future to provide user a leverage to set up her own set of predefined importable KB?

Reply:
> @reckart do we see a use case in future to provide user a leverage to set up her own set of predefined importable KB?

Consider a research group deploying an INCEpTION instance for internal use. The group has some internal KB. It would be nice if they could put these KBs into a configuration file such that users can simply select them from the list when they start a new project instead of having to enter all the details manually.

But IMHO allowing users to set up a second user-defined configuration file or alternatively adding custom KB profiles to settings.properties would be the subject of an separate issue.